PluralKit.Bot/Utils/ContextUtils.cs

```diff
@@ -161,7 +161,7 @@ async Task<int> PromptPageNumber()
                         }
                         catch (TimeoutException)
                         {
-                            await ctx.Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?");
+                            await ctx.Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?", null, null, true);
```

Comment:
This should reference the "what page would you like to go to" message, rather than the original command